usaspending_api/download/management/commands/delta_downloads/award_financial/columns.py

```diff
@@ -0,0 +1,80 @@
+from usaspending_api.download.v2.download_column_historical_lookups import query_paths
+
+groupby_cols = [
```

Comment:
Probably good to have these variables OR the file names call out that this falls under Federal Account. You may be planning to do this in the Treasury Account follow up work, in which case you can ignore this.